meta-ros2-rolling/recipes-bbappends/rosbag2/rosbag2-compression/0001-CMakeLists.txt-drop-dependency-on-zstd_vendor.patch

```diff
@@ -38,12 +38,34 @@ index 176e71ff..e76ab41b 100644
  # Causes the visibility macros to use dllexport rather than dllimport.
  # This is appropriate when building the dll but not consuming it.
  target_compile_definitions(${PROJECT_NAME}_zstd
-@@ -86,7 +86,7 @@ ament_export_include_directories(include)
+@@ -86,7 +86,26 @@ ament_export_include_directories(include)
  ament_export_libraries(${PROJECT_NAME} ${PROJECT_NAME}_zstd)
  ament_export_targets(export_${PROJECT_NAME})
  # order matters here, first vendor, then zstd
 -ament_export_dependencies(pluginlib rcpputils rcutils rosbag2_cpp rosbag2_storage zstd_vendor zstd)
-+ament_export_dependencies(pluginlib rcpputils rcutils rosbag2_cpp rosbag2_storage zstd)
++# don't export zstd as we don't provide zstd from zstd-vendor and rosbag2-{py,transport} would fail with:
```

Comment:
> don't export zstd

But I see `zstd` in `ament_export_dependencies()`. Perhaps you mean `zstd_vendor`?

Reply:
In the removed ament_export_dependencies() here, but not on the line 65 bellow this comment, right?

Reply:
Sorry, I was confused by GitHub's incorrect colorization of the change.